sdk/include/wdf/kmdf/1.9/wdfassert.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * wdfassert.h
+ *
+ * Windows Driver Framework - Prototypes for Runtime Asserts
+ *
+ * This file is part of the ReactOS WDF package.
+ *
+ * Contributors:
+ *   Created by Benjamin Aerni <admin@bennottelling.com>
+ *
+ * Intended Usecase:
+ *   Kernel mode drivers
+ *
+ * THIS SOFTWARE IS NOT COPYRIGHTED
+ *
+ * This source code is offered for use in the public domain. You may
+ * use, modify or distribute it freely.
+ *
+ * This code is distributed in the hope that it will be useful but
+ * WITHOUT ANY WARRANTY. ALL WARRANTIES, EXPRESS OR IMPLIED ARE HEREBY
+ * DISCLAIMED. This includes but is not limited to warranties of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ */
+#ifndef _WDFASSERT_H_
+#define _WDFASSERT_H_
+
+
+#if (NTDDI_VERSION >= NTDDI_WIN2K)
+
+/* RtlAssert isn't defined in XP and Win2k headers so we do so here */
+#ifndef _RTLFUNCS_H
+NTSYSAPI
+VOID
+RtlAssert
+(
```

Comment:
I'm not a fan of this code styling. The parenthesis should go next to the name of the function on the same line.

Reply:
IIRC I did this so it would be more obvious that I didn't copy paste :P 
I'll change everything over though